core/src/main/java/cz/xtf/core/waiting/Waiter.java

```diff
@@ -67,6 +67,30 @@
 	 */
 	Waiter logPoint(LogPoint logPoint);
 
+	/**
+	 * Sets waiters successful awaiting execution.
+	 *
+	 * @param runnable code to be executed upon successful waiting.
+	 * @return this
+	 */
+	Waiter onSuccess(Runnable runnable);
```

Comment:
adding new methods to interfaces is an API breaking change, can we at least make these _default_ ?  (alternatively, we could introduce a new interface extending the original Waiter )

Reply:
(or maybe not even add this into the interface at all, and keep then only in the implementation of NotSoSimpleWaiterAnymore  for now... )